package.json

```diff
@@ -1,36 +1,36 @@
 {
-    "author": "David P. Janes <davidjanes@iotdb.org> (https://iotdb.org)", 
+    "author": "David P. Janes <davidjanes@iotdb.org> (https://iotdb.org)",
     "bugs": {
         "url": "https://github.com/dpjanes/homestar-wemo/issues"
-    }, 
+    },
     "dependencies": {
-        "iotdb": ">=0.4.16", 
+        "iotdb": ">=0.4.16",
         "iotdb-upnp": ">=0.0.1"
-    }, 
-    "description": "HomeStar / IOTDB Bridge for Belkin WeMo", 
+    },
+    "description": "HomeStar / IOTDB Bridge for Belkin WeMo",
     "devDependencies": {
-        "grunt": "~0.4.5", 
-        "grunt-contrib-jshint": "~0.10.0", 
-        "grunt-contrib-nodeunit": "~0.2.0", 
-        "grunt-contrib-uglify": "~0.5.0", 
-        "grunt-contrib-watch": "~0.5.0", 
-        "grunt-jsbeautifier": "^0.2.7", 
-        "jshint-stylish": "~0.1.3", 
-        "load-grunt-tasks": "~0.2.0", 
+        "grunt": "~0.4.5",
+        "grunt-contrib-jshint": "~0.10.0",
+        "grunt-contrib-nodeunit": "~0.2.0",
+        "grunt-contrib-uglify": "~0.5.0",
+        "grunt-contrib-watch": "~0.5.0",
+        "grunt-jsbeautifier": "^0.2.7",
+        "jshint-stylish": "~0.1.3",
+        "load-grunt-tasks": "~0.2.0",
         "request": "^2.36.0"
-    }, 
-    "homepage": "https://homestar.io", 
-    "homestar": {}, 
-    "license": "Apache-2.0", 
-    "main": "index.js", 
-    "name": "homestar-wemo", 
-    "private": false, 
+    },
+    "homepage": "https://homestar.io",
+    "homestar": {},
+    "license": "Apache-2.0",
+    "main": "index.js",
+    "name": "homestar-wemo",
+    "private": false,
     "repository": {
-        "type": "git", 
+        "type": "git",
         "url": "https://github.com/dpjanes/homestar-wemo"
-    }, 
+    },
     "scripts": {
         "test": "grunt"
-    }, 
-    "version": "0.0.5"
+    },
+    "version": "0.0.6"
```

Comment:
Remove tailing space by my editor, and bump version number so we can push it to npm with updated version number